swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+import uuid 
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, category=""):
+        self.id = id if id is not None else uuid.uuid4().int
+        self.condition = condition
+        self.category = category
+    
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type {self.__class__.__name__} with id {self.id}."
+    
+    def condition_description(self):
+        descriptions = {
+        0 : "oof",
+        1 : "been around the block!",
+        2 : "decent",
+        3 : "moderate wear",
+        4 : "gently used",
+        5 : "like new"
+        }
+        if self.condition < 0 or self.condition > 5:
+            return "unknown condition"
+        return descriptions[self.condition]
```

Comment:
If we want coverage on when our method is passed a value that isn't a key in our dictionary, we can use `.get`. It's second argument is what to return if the key isn't found inside of the dictionary that called it. It will also make our code more concise. 
```suggestion
        return descriptions.get(self.condition, "unknown condition")
```